Content.Shared/Salvage/Magnet/SalvageOffering.cs

```diff
@@ -7,5 +7,5 @@ public record struct SalvageOffering : ISalvageMagnetOffering
 {
     public SalvageMapPrototype SalvageMap;
 
-    uint ISalvageMagnetOffering.Cost => 1000; // DeltaV: Station gets next to no benefit from you pulling wrecks, force you to mine first.
+    uint ISalvageMagnetOffering.Cost => 500; // DeltaV: was 1000 (from deltanedas), lowered to 500 to make it less of a grind
```

Comment:
No need to make a comment for old DeltaV changes
```suggestion
    uint ISalvageMagnetOffering.Cost => 500; // DeltaV: Station gets next to no benefit from you pulling wrecks, force you to mine first.
```